pkg/controllers/dynakube/activegate_test.go

```diff
@@ -100,7 +100,7 @@ func TestReconcileActiveGate(t *testing.T) {
 		err := controller.reconcileActiveGate(ctx, dk, nil, nil)
 		require.NoError(t, err)
 	})
-	t.Run(`reconcile automatic kubernetes api monitoring`, func(t *testing.T) {
+	t.Run("reconcile automatic kubernetes api monitoring", func(t *testing.T) {
```

Comment:
i posted in slack discussion, could you please revert this change, and we will fix it in 1 big PR